code/harry_potter_store/test/test_store.py

```diff
@@ -0,0 +1,75 @@
+import unittest
+
+from harry_potter_store.src.store import Store
+
+
+class TestMain(unittest.TestCase):
+
+    def test_can_create_store(self):
+        Store()
+
+    def test_store_contain_five_book_types(self):
+        self.assertEqual(len(Store.book_types), 5)
+
+    def test_get_price_1(self):
+        store = Store()
+        self.assertEqual(store.get_price({"1": 1}), 8)
+
+    def test_get_price_2(self):
+        store = Store()
```

Comment:
This line gets quite repetitive, what about looking at `setUp`/`setUpClass` method?